distribution/lib/Standard/Base/0.0.0-dev/src/Data/Statistics/Rank_Method.enso

```diff
@@ -1,22 +0,0 @@
-
-# TODO Dubious constructor export
-from project.Data.Statistics.Rank_Method.Rank_Method import all
-from project.Data.Statistics.Rank_Method.Rank_Method export all
-
-## Specifies how to handle ranking of equal values.
```

Comment:
Most of the time recently we were moving types to be in their separate file. Is there any reason why here it has been the reverse?

Reply:
As there were so interlinked I put them in the same file. But am happy to break them out later if we wish.

Reply:
Ok, I think I'm happy with keeping it together, especially since its a very simple datatype that does not mean a lot on its own.

Just thinking that we should try to have some general rule of thumb when we want to keep separate and when it can be in a single file - so that when someone else is adding new code (e.g. me), we'll know what to do. I guess the rule is that most types we want to keep separate ('default') but some related helper types may be in the same file?